README.md

```diff
@@ -16,11 +16,18 @@ The top level makefile can be used to build the whole stack.
 
 ## Release Files
 
-In order to keep the release files consistent between the repositories, the makefile
+In order to keep the release files consistent between the repositories, the `Makefile`
 target `release` is used to propagate the paths into the relevant repositories. 
 
 ## Getting started
 
+Clone the repository:
+```
+git clone https://github.com/NSLS-II/epics-modules
+cd epics-modules/
+git submodule init && git submodule update
```

Comment:
Just realized that `make update` takes care of it. Is it worth to remove the line?

Reply:
I think so @mrakitin the `make update` runs this and also stashes any changes, one of the unfortunate things is that the `configure/RELEASE` files are all committed to the repos, so in the act of building you make the repository dirty.

Reply:
I removed the line `git submodule init && git submodule update` yesterday. See the updated code in the "Files changed" tab.